data-prepper-api/build.gradle

```diff
@@ -13,9 +13,9 @@ plugins {
 
 }
 dependencies {
-    implementation "io.micrometer:micrometer-core:1.7.2"
+    implementation 'io.micrometer:micrometer-core'
```

Comment:
I suppose now it is in sync with root dependency io.micrometer:micrometer-bom:1.7.3? Good to know BOM does the centralization.

Reply:
There is a difference between Micrometer and the other three. I failed to mention it in the description above. Because `data-prepper-api` is using Micrometer, I decided to include the BOM for it in under `subprojects`, so `data-prepper-api` gets it.

The other three BOMs are all in a section that excludes `data-prepper-api`.